test/render-component-test.ts

```diff
@@ -0,0 +1,19 @@
+import buildApp from './test-helpers/test-app';
+
+const { module, test } = QUnit;
+
+module('renderComponent');
+
+test('can render a component', function(assert) {
+  assert.expect(1);
+
+  let containerElement = document.createElement('div');
+
+  let app = buildApp()
+    .template('hello-world', `<h1>Hello Glimmer!</h1>`)
+    .boot();
+
+  return app.renderComponent('hello-world', containerElement).then(() => {
+    assert.equal(containerElement.innerHTML, '<h1>Hello Glimmer!</h1>');
+  });
+});
```

Comment:
Can you add tests for rendering multiple components, rendering into tags with existing content (with or without nextSibling), etc?